ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/PrivacyBlockProcessor.java

```diff
@@ -100,7 +102,13 @@ public BlockProcessingResult processBlock(
 
     final BlockProcessingResult result =
         blockProcessor.processBlock(
-            blockchain, worldState, blockHeader, transactions, ommers, metadataUpdater);
+            blockchain,
+            worldState,
+            blockHeader,
+            transactions,
+            ommers,
+            withdrawals,
```

Comment:
To play 'mainnet devil's advocate', what's the reasoning for changing PrivacyBlockProcessor as well?

Reply:
That it wouldn't compile without this change as I updated the full arg processBlock method to include Withdrawals. Would prefer not to touch this since it's going to be removed soon. But it's also a trivial change.